packages/logging/src/servicies/logger/logger.service.spec.ts

```diff
@@ -0,0 +1,122 @@
+import createHttpError from "http-errors";
+import pino from "pino";
+import pinoFluentd from "pino-fluentd";
+import pretty from "pino-pretty";
+
+import { config } from "../../config";
+import { Logger, LoggerService } from "./logger.service";
+
+jest.mock("pino");
+jest.mock("pino-fluentd");
+jest.mock("pino-pretty");
+
+describe("LoggerService", () => {
+  let loggerService: LoggerService;
+  let mockLogger: jest.Mocked<Logger>;
+  const defaultLogFormat = config.STD_OUT_LOG_FORMAT;
+  const defaultFluentdTag = config.FLUENTD_TAG;
+  const defaultFluentdHost = config.FLUENTD_HOST;
+  const defaultFluentdPort = config.FLUENTD_PORT;
+
+  beforeEach(() => {
+    mockLogger = {
+      info: jest.fn(),
+      error: jest.fn(),
+      warn: jest.fn(),
+      debug: jest.fn(),
+      child: jest.fn().mockReturnThis()
+    } as unknown as jest.Mocked<Logger>;
+
+    (pino as unknown as jest.Mock).mockReturnValue(mockLogger);
+    loggerService = new LoggerService();
+  });
+
+  afterEach(() => {
+    config.STD_OUT_LOG_FORMAT = defaultLogFormat;
+    config.FLUENTD_TAG = defaultFluentdTag;
+    config.FLUENTD_HOST = defaultFluentdHost;
+    config.FLUENTD_PORT = defaultFluentdPort;
+    jest.clearAllMocks();
+  });
+
+  describe("constructor", () => {
```

Comment:
ouch, I've just noticed there's the actual method `initPino`. So the describe you had before actually makes more sense! apologies!
```suggestion
  describe("prototype.initPino", () => {
```

If you don't want to change it, I'll just do it myself later :)